src/main/java/nextstep/subway/line/domain/SectionConnectManager.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.subway.line.domain;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import static java.util.stream.Collectors.toSet;
+
+public class SectionConnectManager {
+    public static void connectAll(Line line, Section request, List<Section> matchedSections) {
+        Set<Section> requests = matchedSections.stream()
+                .map(section -> SectionConnectManager.connect(section, request))
+                .filter(Objects::nonNull)
+                .collect(toSet());
+
+        addNewSections(line, requests);
+    }

```

Comment:
`matchedSections`는 `request`에 의해 변경되고
`request`는 `line`에 추가되면 될것같은데

```suggestion
    public static void connectAll(Line line, Section request, List<Section> matchedSections) {
        matchedSections.forEach(section -> SectionConnectManager.connect(section, request));

        addNewSection(line, request);
    }

```
이렇게 하는건 어떤가요? 😀

Reply:
앗 그러네요..! 굳이 왜 리스트로 받아서 처리하려했는지.. 감사합니다 ㅎㅎㅎ